kata-deploy/README.md

```diff
@@ -15,7 +15,7 @@
       - [Kata deploy](#kata-deploy)
       - [Kata cleanup](#kata-cleanup)
 
-[kata-deploy](kata-deploy) provides a Dockerfile, which contains all of the binaries
+[kata-deploy](.) provides a Dockerfile, which contains all of the binaries
```

Comment:
strangely self referential...

Reply:
Yep ;) That will link to the current directory and show the files that comprise "kata-deploy" which seems to be what the link was trying to do originally.